### PR TITLE
role form table created to list and update roles #1

### PR DESCRIPTION
**Pull request by shubhamsingh3211**
## Description
role form table created to list and update roles
action button is added to create new roles

### CODE REVIEW

**Comment by shubhamsingh3211** [comment]
717 additions are of  package-lock.json

File: src/pages/settings/AddRole/AddRoleView.tsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ function AddRoleView() {
 				<Grid item xs={12}>
 					<Paper>
 						<Box p={2}>
-							<AddRoleForm />
+
```
Comment (ShadabFaiz) [inline src/pages/settings/AddRole/AddRoleView.tsx:22]: If you have removed `AddRoleForm` component, then what is the point of keep `Box` component?

**Review by ShadabFaiz — COMMENTED** [review]
Why there is `package-lock.json` file but not `package.json`?

**Review by ShadabFaiz — APPROVED** [review]
LGTM

**Pull request merged**